### PR TITLE
Initial setup of sample repository

### PR DESCRIPTION
**Pull request by pattiereaves**
This still has some work to do and documentation to add, but it is a fresh start: The Huron devserver and build tasks are working.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Logs
```
Comment (pattiereaves) [inline .gitignore]: This was generated by `npm init`, should we keep all of it?

File: huron.config.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+path = require('path');
```
Comment (pattiereaves) [inline huron.config.js]: @todo: needs documentation.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "huron-example-setup",
+  "version": "1.0.0",
+  "description": "An example site using the Huron prototyping tool.",
+  "main": "index.js",
+  "scripts": {
+    "//" : "The Huron script runs the devserver so you can build prototypes on the fly",
```
Comment (pattiereaves) [inline package.json]: This is how you're supposed to comment build scripts, I guess?

File: scss/prototypes/prototype-sample.html [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+	<ul>
+		<li>Name it in the <code>prototypes</code> array in your huron.config.js.</li>
+		<li>Create a <code>prototypes</code> folder inside one of your KSS folders (that also should listed in your Huron config).</li>
+		<li>Name the prototype file should be <code>prototype-[name].html</code>.</li>
+		<li>Make sure to wrap the contents of the prototype with a single root element (e.g. <code>div</code>).</li>
+	</ul>
+
+	<hr>
+
+	<h2>Example KSS partials:</h2>
```
Comment (pattiereaves) [inline scss/prototypes/prototype-sample.html]: @todo: This should be documented better.

File: package.json (hunk 3 above)
Comment (pattiereaves) [inline package.json]: Also we can update this once your CLI build is released 💃

File: .gitignore (hunk 1 above)
Comment (ostowe) [inline .gitignore]: We can keep for now, though not sure we'll need all of it

File: huron.config.js (hunk 2 above)
Comment (ostowe) [inline huron.config.js]: Maybe we just direct users to the `huron` config docs?

File: package.json (hunk 3 above)
Comment (ostowe) [inline package.json]: Interesting, alright! Might be clearer to document recommended scripts in the `README.md`

File: scss/prototypes/prototype-sample.html (hunk 4 above)
Comment (ostowe) [inline scss/prototypes/prototype-sample.html]: We could probably direct people to the `huron` docs here as well, though probably more we could add in this repo in terms of best practices and ideas.

**Pull request merged**